src/services/item/plugins/thumbnail/service.ts

```diff
@@ -64,4 +65,21 @@ export class ItemThumbnailService {
 
     return result;
   }
+
+  async deleteAllThumbnailSizes(
+    actor: Member,
+    repositories: Repositories,
+    { itemId }: { itemId: string },
+  ) {
+    const item = await repositories.itemRepository.get(itemId);
+    await validatePermission(repositories, PermissionLevel.Write, actor, item);
```

Comment:
Use `this.itemService.get(itemId, PermissionLevel.Write)` as it does the validation.